WorkAndFun/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!--/* Bootstrap CSS */-->
+    <link th:href="@{/css/bootstrap.min.css}" rel="stylesheet"/>
+
+    <link th:href="@{/css/style.css}" rel="stylesheet"/>
+    <title th:text="#{index.title}"></title>
+</head>
+
+<body class="bg-light p-3">
+<div class="container-lg">
+    <div class="row mb-1">
+        <div class="col-md-12" th:insert="~{fragments/navbar :: nav}">
+        </div>
+</div>
+        <!-- JUMBOTRON -->
+      <div class="row">
+        <div class="col-md-12">
+            <div class="jumbotron text-center"
+                 th:style="'background: url(img/drillguy.jpg) no-repeat; background-size: 1150px; height: 350px'">
+                <h2 class="mt-5 white-text-shadow">
+                    Zamawianie usług online
+                </h2>
+                <p class="white-text-shadow">
+                    Ponad 100 000 sprawdzonych profesjonalistów do wykonywania zadań
+                </p>
+                <form class="form-inline justify-content-center">
+                    <input class="form-control mr-sm-2" type="text">
+                    <button class="btn btn-primary my-2 my-sm-0" type="submit">
+                        Szukaj
+                    </button>
+                </form>
+            </div>
+        </div>
+    </div>
+
+    <!-- KATEGORIE-->
+    <div class="row">
+        <div class="col-md-12 mt-2">
+            <ul class="list-group list-group-horizontal-md list-group-flush">
+                <li class="flex-fill list-inline-item text-center list-group-item-action bg-default">
```

Comment:
Troszkę mi się nie podoba osobiście, że jest tu sporo powtarzających się pozycji, zwłasza że do każdej powtarza się klasa, width-height. Plus że ręcznie jest podział na 2 rowsy. Najlepiej to ułożyć nie listą (< li >) tylko div-ami, i nadać każdemu klasę col (automatyczna szerokość). Bo chyba bootstrap zdecydowanie lepiej działą na div-ach, jeśli je opakowujemy siatką grid.
Ale ze względu na ograniczenia czasowe jakie mamy, to tego nie będę się czepiał, skoro działa, tylko zaznaczam że troszkę to moim zdaniem do dopracowania, gdybyśmy mieli więcej czasu.